jetson/nav/search/spiral_search_points.txt

```diff
@@ -0,0 +1,14 @@
+0.0 90.0

```

Comment:
add a comment in the read me for this saying what the numbers in this file mean (like what the colomns are)